README.md

```diff
@@ -28,6 +28,8 @@ __Note:__ Mac OS X, Linux, and Windows are currently the only supported OSes.
       * ```sudo systemctl stop bluetooth``` (once)
       * ```sudo systemctl disable bluetooth``` (persist on reboot)
 
+If you're using BlueZ 5.14 or later, you may find that running noble *and* bleno at the same time stops you from retrieving the list of services over BLE.
+
```

Comment:
Ahh, that looks great. Can't believe I missed that (I must admit I only looked in Bleno's docs, not Noble - but I guess others may do the same). I've just pushed a documentation change to link to it - but feel free to move it around or cancel the PR and make your own changes :)